pkg/clusteragent/orchestrator/controller.go

```diff
@@ -116,13 +118,21 @@ func newController(ctx ControllerContext) (*Controller, error) {
 	serviceInformer := ctx.InformerFactory.Core().V1().Services()
 	nodesInformer := ctx.InformerFactory.Core().V1().Nodes()
 
-	cfg := processcfg.NewDefaultAgentConfig(true)
-	if err := cfg.LoadProcessYamlConfig(ctx.ConfigPath); err != nil {
+	// Deprecated, we want to move away from processcfg
+	processCfg := processcfg.NewDefaultAgentConfig(true)
+	if err := processCfg.LoadProcessYamlConfig(ctx.ConfigPath); err != nil {
 		log.Errorf("Error loading the process config: %s", err)
 	}
 
+	// We slowly want to migrate from processcfg to orchestratorCfg
+	orchestratorCfg := orchcfg.NewDefaultOrchestratorConfig()
```

Comment:
I tried to started the move out of processCfg type for orchestrator related settings. To support interoperability I use both for now.